pkg/common/common.go

```diff
@@ -0,0 +1,119 @@
+package common
```

Comment:
name does not sound right :/ Maybe we should move all this operator to subpackage like:
```
pkg/operators/rbac/common
pkg/operators/rbac/controllernamespaces
...
``` 

@kargakis @jim-minter wdyt?

Reply:
I agree with the common being out of place.  A self containing option would be prefered.  I'll defer to the more seasoned go veterans for pkg help.